zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterOption.java

```diff
@@ -21,25 +21,25 @@
  *
  */
 public class InterpreterOption {
-  boolean remote;
+//  boolean remote;
```

Comment:
There're some corresponding parts in front-end side. for example https://github.com/apache/zeppelin/blob/master/zeppelin-web/src/app/interpreter/interpreter.controller.js#L123